ios/Classes/ComAppceleratorUrlSessionSessionProxy.m

```diff
@@ -25,6 +25,15 @@ - (id)_initWithPageContext:(id<TiEvaluator>)context andArguments:(NSDictionary*)
     
     return self;
 }
+-(void)_initWithProperties:(NSDictionary *)properties
+{
+    sessionConfiguration = [properties objectForKey:@"configuration"];
+    [self rememberProxy:sessionConfiguration];
+    session = [NSURLSession sessionWithConfiguration:[sessionConfiguration sessionConfiguration]
+                                            delegate:(id<NSURLSessionDelegate>)[[UIApplication sharedApplication] delegate]
+                                       delegateQueue:nil];
+    [super _initWithProperties:properties];
```

Comment:
This whole logic has already been done in `_initWithPageContext:andArguments` above, why would you do it here as well?